packages/components/src/mobile/bottom-sheet/range-cell.native.js

```diff
@@ -20,7 +20,7 @@ export default function BottomSheetRangeCell( props ) {
 		step = 1,
 		minimumTrackTintColor = '#00669b',
 		maximumTrackTintColor = Platform.OS === 'ios' ? '#e9eff3' : '#909090',
-		thumbTintColor = Platform.OS === 'ios' ? '#fff' : '#00669b',
+		thumbTintColor = Platform.OS === 'android' && '#00669b',
```

Comment:
I've noticed that if `thumbTintColor` on iOS is set to white color it losing the shadow on iPhoneX, however it works as expected on other iPhones. I've decided to use the custom color 

with white color | with default color
--- | ---
![IMG_0008](https://user-images.githubusercontent.com/22746080/67198083-7fa0d400-f3fe-11e9-8a13-04dace43e1ce.PNG) | ![IMG_0007](https://user-images.githubusercontent.com/22746080/67198087-80d20100-f3fe-11e9-81c7-1f3283de7ed1.PNG)

Currently we are using the `Slider` from react-native, which will be deprecated soon and we'll have to use the `Slider` from community library. Need to check if the bug doesn't exist in that library.

Reply:
I've tested this using slider library from a community and it seems like a bug on ios version ~12.

Tested on:
* device - iPhone X - 12.3.1 - doesn't work properly
* device - iPhone X - 13.1.3 - works as expected
* device - iPhone 7 - 12.4 -  doesn't work properly
* device -  iPhone 7 - 13.1.3  - works as expected